content/ja/docs/collector/quick-start.md

```diff
@@ -0,0 +1,129 @@
+---
+title: クイックスタート
+cSpell:ignore: docker dokey dpkg okey telemetrygen
+description: コレクターをセットアップとテレメトリーの収集をすぐに始めてみましょう！
+weight: 1
+cSpell:ignore: gobin
```

Comment:
There should be only one `cSpell:ignore` front matter key.

Reply:
Also, remove any keys that don't apply anymore.

Reply:
@chalin thank you for reviewing. I removed unused fields from metadata.